web/themes/custom/theme_cat/templates/paragraphs/paragraph--image-and-text--default.html.twig

```diff
@@ -0,0 +1,12 @@
+{% embed "@molecules/card-test/card-test.twig" with {
```

Comment:
for fields that are not required, could you add a check to maje sure they only render when not empty?